### ISSUE TITLE
asterisk-chan-dongle: no permission to access the device

### ISSUE THREAD

**Issue by 532910** [openwrt/telephony#684]
Maintainer: @jslachta
Environment: cortexa53 friendlyarm_nanopi-neo2 21.02.0

Description:
1. Starting with 21.02.0 asterisk runs as `asterisk` user.
2. `/dev/ttyUSB*` has `660` `root:dialout` permissions.
3. There is no way to change group automatically (at boot-up or device plug for example).
4. Adding asterisk to `dialout` group gives nothing.

**Comment by micmac1** [comment]
See my comment in #685

**Comment by rojer** [comment]
workaround - add the following hotplug config:
```
root@SP-BR1:~# cat /etc/hotplug.d/usb/30-ttyUSB
chmod 1777 /var/lock
chown asterisk /dev/ttyUSB*
```

**Comment by 532910** [comment]
Possible workaround but not a solution of couse.

**Comment by micmac1** [comment]
Backported fixes to 21.02. Closing issue.

Thanks!
